router.go

```diff
@@ -93,6 +93,9 @@ func (r *Router) Add(method, path string, h HandlerFunc) {
 
 	if h == nil && r.echo.Logger != nil {
 		// FIXME: in future we should return error
+		if r.echo.Logger == nil {
```

Comment:
This does not make sense to me. Two lines up there is `r.echo.Logger != nil` and now we are here checking if it is nil.

Reply:
Sorry, have fixed:)